source/cps/index.rst

```diff
@@ -0,0 +1,29 @@
+CPS
+===
+
+:软件名称: CPS
```

Comment:
"CPS" or "Computer Programs in Seismology"?

Reply:
I prefer CPS because I think all know CPS instead of its full name, just like SAC.

Reply:
OK.